### PR TITLE
Petri termoverview

### PR DESCRIPTION
**Pull request by Petrikow**
This should be close to final in terms of terminology.

That said, there are some edits in here too, especially for the Vlov scene where Arc speaks Hime-mode. Since I was the one who went to over-the-top last time editing it, I felt like I should probably reign myself in a little bit, no matter how much I like poetic expressions.

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt [hunk 1]
```diff
@@ -454,7 +454,7 @@ and those who came after are the young blood."
 -- Page 605, Offset 18092.
 -- 吸血鬼の尺度で言うと、ヴローヴはつい最近祖になった死徒である、と。
 From a vampire's perspective,
- Vlov was a Dead Apostle that had only recently become an Ancestor.
+ Vlov is a Dead Apostle that has only recently become an Ancestor.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:457]: I'd keep past tense, he's dead at this point (I assume??)

File: script/Arcueid/Day 8/08_00_ARC08_7.txt [hunk 2]
```diff
@@ -265,7 +265,7 @@ It would be strange to assume that you were the only special case within the ent
 [sha:8779bed4a71332cfc9ba638c746e3ef3504db20b]{
 -- Page 445, Offset 20829.
 -- あなたに異能があるのなら―――あなたの家系すべてが特別、と考えるべき。志貴の家族ってそういうヒトたちなんじゃないの？」
-Rather, if you have a power, then you should conclude that it's something that runs in the family.
+In fact, if you have an ability, then its easy to conclude that it's something that runs in your family.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 8/08_00_ARC08_7.txt:268]: > then **it's** easy to conclude

**Review by rschlaikjer — APPROVED** [review]
otsukare

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 3]
```diff
@@ -311,22 +311,22 @@ But before that, allow me to extend to you at least this courtesy."
 [sha:81dfc32aae41ec3d7ea19bdd24a21b8ce27ccf23]{
 -- Page 539, Offset 10041.
 -- 「“―――答えよ、矮小家畜の我が隷属。
-"―――Heed me, meagre chattel, of me enslaved.
+"'―――Heed me, mine enslaved and meagre chattel.
```
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:314]: No, simulating the effect of the game.

Look, there are “ inside the brackets. Arcueid is speaking using Brunestud's voice.

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt (hunk 1 above)
Comment (Petrikow) [reply script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:457]: True, lost the context there for a second.

File: script/Arcueid/Day 12/12_00_ARC12_2B.txt [hunk 4]
```diff
@@ -398,7 +398,7 @@ Even through his microphone, I can sense Mario's disappointment. His despair. Hi
 [sha:b874a8bc61b39a4314e9a0ea1cb29d14fcc4ae9e]{
 -- Page 102, Offset 30829.
 -- 「そう言った。アレらは私が開発した魔術基盤。
-"I did. These are all %{i}Magecraft Foundations%{/i} that I have developed.
+"I did. These are all %{i}magecraft foundations%{/i} that I have developed.
```
Comment (yk1512) [inline script/Arcueid/Day 12/12_00_ARC12_2B.txt:401]: All the other cases of these compound magecraft terms are capitalised below.

File: script/Arcueid/Day 4/04_00_ARC04_7_1.txt [hunk 5]
```diff
@@ -68,7 +68,7 @@ Arcueid falls silent, a serious expression on her face.
 [sha:c9aaa42138d8517b79c7004e171229be16950812]{
 -- Page 644, Offset 7286.
 -- 「知ってるわ。現存する四人の魔法使いの一人だもの。
-"I know of her. She's one of the four Magicians that exist in this world.
+"I know of her. She's one of the four extant %{i}Magicians%{/i}.
```
Comment (yk1512) [inline script/Arcueid/Day 4/04_00_ARC04_7_1.txt:71]: extant is such a cool word

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 6]
```diff
@@ -510,12 +510,12 @@ And then―――
 [sha:5a34da987dbe2ad8e37cd710badd8724996c6c5a]{
 -- Page 551, Offset 10080.
 -- <彼|か>の領土はこの星の一部でありながら、“生きるに<能|あた>わず”と除外された生命圏。
-Although this domain remains of the planet, all the same is it a biosphere foreclosed... in which none may live...
+Although this domain is part of the planet, all the same is it a biosphere foreclosed, in which none can live...
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:513]: it is a biosphere foreclosed all the same,

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt [hunk 7]
```diff
@@ -218,8 +218,8 @@ There are twenty-seven of them, so we call them the %{i}Twenty-Seven Ancestors%{
 [sha:348faeee60a7ce4a41db73c662eda08e39bd919e]{
 -- Page 590, Offset 18052.
 -- Ⅵ階梯のモノが祖に認められ、更なる異能を与えられたもの。同じ呼び名でもその規模はⅥ階梯とは別物よ」
-This Rank consists of those from Rank VI who were recognized by the Ancestors, and granted further powers.
- They retain the same title, but their abilities are in a completely different league to those of Rank VI."
+This Rank consists of those from Rank VI who were recognized by the Ancestors, and granted further abilities.
+ They're in completely different league to Rank VIs, even if they're called by the same name."
```
Comment (yk1512) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:222]: a completely different league.

File: script/Arcueid/Day 8/08_00_ARC08_7.txt [hunk 8]
```diff
@@ -265,7 +265,7 @@ It would be strange to assume that you were the only special case within the ent
 [sha:8779bed4a71332cfc9ba638c746e3ef3504db20b]{
 -- Page 445, Offset 20829.
 -- あなたに異能があるのなら―――あなたの家系すべてが特別、と考えるべき。志貴の家族ってそういうヒトたちなんじゃないの？」
-Rather, if you have a power, then you should conclude that it's something that runs in the family.
+In fact, if you have an ability, then it's easy to conclude that it's something that runs in your family.
```
Comment (yk1512) [inline script/Arcueid/Day 8/08_00_ARC08_7.txt:268]: > then it can be easily concluded that it's

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 9]
```diff
@@ -1015,8 +1015,8 @@ First one's that they gotta be wealthy.
 [sha:83380ef9ab7b6e8944a05c12a6d5112e9d61e885]{
 -- Page 732, Offset 26260.
 -- 魔術のような学んで修得する神秘ではなく、生まれついてその肉体が持ってしまう特異能力。
-And I ain't talkin' about Arcana which you can study, like magecraft,
- but supernatural powers present in the body from the moment it's born.
+And I ain't talkin' about arcana which you gain by studyin', like magecraft,
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:1018]: "you gain" "it's born" mismatch. first comma adds unnecessary pause. Jersey-go:
> And I ain't talkin' about arcana you gain by by studyin' like magecraft, I'm talkin' about special abilities present in the body from the moment you're born.

File: script/Arcueid/Day 12/12_00_ARC12_2B.txt [hunk 10]
```diff
@@ -72,7 +72,7 @@ The one who smashed the third-floor window and came tumbling inside is―――
 -- Page 81, Offset 30767.
 -- 「……数秘門による雷霆ですか。
 // 数秘門 is a typo here, this is supposed to be 数秘紋
-"...Gematria-based lightning. I see.
+"Numeromancy-based lightning... I see.
```
Comment (Avuxo) [inline script/Arcueid/Day 12/12_00_ARC12_2B.txt:75]: numeromancy sounds either super dope or super dope-y and i can't decide which

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 11]
```diff
@@ -470,12 +470,12 @@ It's as though he were grinning in agreement with Arcueid's provocation.
...
 -- Page 548, Offset 10072.
 -- ……貴様はまさにソレだ。そこまで衰退していながら、まだおれを上回る余力がある。
-And that is truly what you are... Though diminished your power may be... above mine it is nonetheless.
+And that is truly what you are... Though your power may be diminished... it is above mine nonetheless.
 }
 [sha:0cf50eb40e3ac194055940c470a381326109bc97]{
 -- Page 548, Offset 10073.
 -- あらゆる呪い、あらゆる状況に対応する原初の<一|いち>―――並の死徒ではかなうまい」
-One who can with all curses deal; through all situations adapt: The Original One... an ordinary Dead Apostle stands no chance..."
+One who can with all curses deal; through all situations adapt: the Original One... an ordinary Dead Apostle stands no chance..."
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:478]: An ordinary

File: script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt [hunk 12]
```diff
@@ -218,8 +218,8 @@ There are twenty-seven of them, so we call them the %{i}Twenty-Seven Ancestors%{
 [sha:348faeee60a7ce4a41db73c662eda08e39bd919e]{
 -- Page 590, Offset 18052.
 -- Ⅵ階梯のモノが祖に認められ、更なる異能を与えられたもの。同じ呼び名でもその規模はⅥ階梯とは別物よ」
-This Rank consists of those from Rank VI who were recognized by the Ancestors, and granted further powers.
- They retain the same title, but their abilities are in a completely different league to those of Rank VI."
+This Rank consists of those from Rank VI who were recognized by the Ancestors, and granted further abilities.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt:221]: drop comma

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 5/05_00_ARC05_8.txt (hunk 6 above)
Comment (Petrikow) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:513]: I feel like the follow up doesn't flow as well then. Is that just me?

> Although this domain is part of the planet, it is a biosphere foreclosed all the same, in which none can live...

File: script/Arcueid/Day 5/05_00_ARC05_8.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:314]: Actually thinking about it more, maybe this is a case were you'd use italics on the whole sentence instead?

[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:314]: We've talked a few times about using a special font for these sections, something gothic-looking. We're a little close to the wire but I can prototype it this evening if you think that'd be a good solution.

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:314]: That sound cool, but yeah it's a bit tight.

Gothic/Fraktur typefaces themselves can be difficult to read, an Antiqua/Roman type might be better.

That said, if that solution can't materialize in time I still think just using italics is probably clear enough.

[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:314]: Only reason why it wouldn't be possible is if i can't work out font forge in time. I'll spend some time on it

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:314]: For now I'll just remove the "" and let you apply this.

File: script/Arcueid/Day 12/12_00_ARC12_2B.txt (hunk 10 above)
Comment (Petrikow) [reply script/Arcueid/Day 12/12_00_ARC12_2B.txt:75]: It's dope

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 13]
```diff
@@ -311,12 +311,12 @@ But before that, allow me to extend to you at least this courtesy."
...
 -- Page 539, Offset 10041.
 -- 「“―――答えよ、矮小家畜の我が隷属。
-"'―――Heed me, mine enslaved and meagre chattel.
+"―――Heed me, mine enslaved and meagre chattel.
 }
 [sha:39a5e235718761df3e6edcf770919a0e44ba26f9]{
 -- Page 539, Offset 10042.
 -- 祖の<証|あかし>。<根|ね>の病巣。大渦の如きその穴蔵。
-The mark of an Ancestor; the root of thy sickness; that cavity like the maelstrom;
+The mark of an Ancestor; the root of thy sickness; that maelstrom-like cavity;
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:319]: could use a comment

File: script/Arcueid/Day 5/05_00_ARC05_8.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:513]: Yeah I think what you already have here is better

File: script/Arcueid/Day 5/05_00_ARC05_8.txt (hunk 13 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:319]: As in "Is this OK?" I guess I have no idea what 穴蔵 is supposed to be referring to here. Is it a crypt? Is it a metaphorical hole? Is it a root cellar where you keep the vampire veggies?

Comment (Petrikow) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:319]: I think it's meant to correspond to the 'hole' Shiki sees in Vlov

Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:319]: Is Arc able to perceive / know the existence of it?

Comment (Petrikow) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:319]: What a great question; somehow I feel like the answer to that isn't quite supposed to be spoken off in a Github comment, but... (keep in mind a lot of this is speculatory in nature)

The Mystic Eyes of Death Perception don't just let Shiki see things that related to 'the fated end' of things. In fact, their proximity to the Root lets Shiki see a lot of strange things he shouldn't. This is perhaps actually related to his latent pre-MEODP ability: the Pure Eyes. The thing he sees in Vlov isn't really his point of death, it actually seems to be his Principle. That said, Principles are so connected to an individual that they appear around the source of their existence, their point of death. 

So really, Arcueid probably can't see it, but she is implicitly asking him: what is your Principle?
All these things she's naming as merely (mark of an ancestor, etc) ways of referring to Vlov's Principle (as is evident in the line after). So this "maelstrom-like cavity" is basically well-known already as the shape of a Principle...

That's my interpretation, at least.

Man, and that's 1/3rd of a line.

**Pull request merged**